src/app/pages/app-bar/dropdown-toolbar/dropdown-toolbar.component.ts

```diff
@@ -0,0 +1,65 @@
+import { AfterViewInit, Component, OnInit, ViewChild } from '@angular/core';
+import { BreakpointObserver, Breakpoints, BreakpointState } from '@angular/cdk/layout';
+import { DropdownToolbarComponent as PxbDropdownToolbar } from '@pxblue/angular-components';
+import { MatBottomSheet } from '@angular/material/bottom-sheet';
+import { DropdownToolbarStateService } from './dropdown-toolbar.service';
+import { DropdownToolbarBottomSheetExample } from './dropdown-toolbar-bottomsheet.component';
+import { StateService } from '../../../services/state.service';
+import { menuItems } from './menuItems';
+
+@Component({
+    selector: 'app-dropdown-toolbar',
+    templateUrl: './dropdown-toolbar.component.html',
+})
+export class DropdownToolbarComponent implements OnInit, AfterViewInit {
+    isSmall: boolean;
+    items = menuItems;
+
+    @ViewChild('toolbar') dropdownToolbar: PxbDropdownToolbar;
+
+    constructor(
+        private readonly _drawerService: StateService,
+        private readonly _breakpointObserver: BreakpointObserver,
+        private readonly _bottomSheet: MatBottomSheet,
+        private readonly _stateService: DropdownToolbarStateService
+    ) {}
+
+    ngOnInit(): void {
+        this._breakpointObserver
+            .observe([Breakpoints.Small, Breakpoints.Handset])
```

Comment:
Can we use `XSmall` here? `Small` is still a bit wide for a bottom sheet.

Reply:
You got it, this is using xsmall now.